LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -104,65 +54,100 @@ public void CreateAccounts(int accountCount)
 
                 Console.WriteLine('\n');
             }
+
+            ReportSummary("Account", accountCount);
+            ReportFooter("Create Account");
         }
 
         public void ActivateAccounts()
         {
-            Console.WriteLine("-----Unfair coordinator started activating accounts-----\n");
+            ReportHeader("Activate account");
 
             while (pendingAccountStack.Count > 0)
             {
                 pendingAccountStack.Pop().Status = Status.Active;
-
-                Console.WriteLine($" Activated at {DateTime.Now}\n");
             }
 
-            Console.WriteLine("-----Unfair coordinator finished activating accounts-----");
+            ReportFooter("Activate account");
         }
 
         public void CreateSubjects(int subjectCount)
         {
-            for (int i = 0; i < subjectCount; i++)
+            ReportHeader("Subject generation");
+
+            for (int i = 1; i <= subjectCount; i++)
             {
                 Subject subject = ObjectGenerator.GenerateSubject();
 
+                ReportItem("Subject", i);
```

Comment:
Combine in one call the following lines:
ReportItem("Subject", i);
subject.Report();